### PR TITLE
Fix webaddress contains dash throw error

### LINKED ISSUE #6: When webAddress contains a dash throws an argument error
When webAddress contains a dash throws an argument error
Example: 
 - www.star-wars.com
 - demo-api.starwars.com

image

### PR DESCRIPTION
**Pull request by fernandezja**
Fix #6 
Add few tests to verify
image

### CODE REVIEW

File: HttpPing/Interfaces/IEnvironmentService.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HttpPing.Interfaces
+{
+    internal interface IEnvironmentService
+    {
+        bool IsExit { get; }
```
Comment (Killeroo) [inline HttpPing/Interfaces/IEnvironmentService.cs:11]: I think `HasExited` is a clearer name

File: HttpPing/Program.cs [hunk 2]
```diff
@@ -28,13 +32,24 @@ class Program
 
         private static string resolvedAddress = "";
 
-        private static void Main(string[] args)
+        private static IEnvironmentService EnvironmentService;
+
+
+        /// <summary>
+        ///
```
Comment (Killeroo) [inline HttpPing/Program.cs:39]: Either remove or fill in this xml comment

File: HttpPing/Program.cs [hunk 3]
```diff
@@ -104,9 +120,14 @@ private static void Main(string[] args)
...
                         default:
-                            if (arg.Contains("-"))
-                                throw new ArgumentException();
+                            if (count > 0)
+                            {
+                                if (arg.Contains("-"))
+                                    throw new ArgumentException();
+                            }
                             break;
+
```
Comment (Killeroo) [inline HttpPing/Program.cs:129]: Unneccessary empty lines

File: HttpPing/Program.cs [hunk 4]
```diff
@@ -171,6 +195,17 @@ private static void Main(string[] args)
             // Results?
         }
 
+
+        /// <summary>
+        ///
```
Comment (Killeroo) [inline HttpPing/Program.cs:200]: Again fill out or remove this comment

**Review by Killeroo — COMMENTED** [review]
Good changes, thanks for adding tests as well, really useful.

Left some very minor comments.

File: HttpPing/Interfaces/IEnvironmentService.cs (hunk 1 above)
Comment (fernandezja) [reply HttpPing/Interfaces/IEnvironmentService.cs:11]: Yes you're right

**Comment by Killeroo** [comment]
Looks good, thanks again!

**Pull request merged**